### DISCUSSION TITLE
Supporting additional algorithms

### DISCUSSION THREAD

**Discussion by GregDomjan** [Yubico/libfido2#747] [Q&A]
Support for Cose ES384 algorithm was added (recently? ~1.11)

The Python library includes support for additional algorithms ES512, PS256, ES256K
https://github.com/Yubico/python-fido2/commit/79b74aa3839dea9285ca869b159ef61b256033e7
https://github.com/Yubico/python-fido2/commit/c8fd18d4b242d895670014cff4014934d5ce01f4

Is similar support in libfido2 on a roadmap?

**Comment by LDVG** [comment]
Hi,

Generally speaking, we add support for algorithms in libfido2 when we know that there are authenticators out there that also implements them. I have personally not seen any authenticators with support for these yet. If anyone does, please let us know.
